wire-library/wire-runtime/build.gradle.kts

```diff
@@ -138,3 +139,26 @@ configure<MavenPublishBaseExtension> {
     KotlinMultiplatform(javadocJar = Dokka("dokkaGfm"))
   )
 }
+
+val generateVersion by tasks.creating {
+  val outputDir = file("src/generated/kotlin")
+
+  inputs.property("version", version)
+  outputs.dir(outputDir)
+
+  doLast {
+    val versionFile = file("${outputDir}/com/squareup/wire/Version.kt")
+    versionFile.parentFile.mkdirs()
+    versionFile.writeText(
+        """
+      |// Generated file. Do not edit!
+      |package com.squareup.wire
+      |const val VERSION = "${project.version}"
```

Comment:
I don’t think this should be const.
https://kotlinlang.org/docs/whatsnew11.html#constant-inlining
https://kotlinlang.slack.com/archives/C0922A726/p1526398431000786

Reply:
https://github.com/square/okhttp/pull/6947